cmd/trace-agent/config/config.go

```diff
@@ -80,7 +82,8 @@ func prepareConfig(path string) (*config.AgentConfig, error) {
 	}
 	cfg.ConfigPath = path
 	if coreconfig.Datadog.GetBool("remote_configuration.enabled") && coreconfig.Datadog.GetBool("remote_configuration.apm_sampling.enabled") {
-		if client, err := newRemoteClient(); err != nil {
+		client, err := remote.NewClient("trace-agent", version.AgentVersion, []data.Product{data.ProductAPMSampling}, time.Second*1)
```

Comment:
I think the time.Second*1 should be a global var in this file so its meaning is clearer at a quick glance. In the comment for that var you can describe why 1 second is the right choice (why not higher or lower).